package.json

```
@@ -39,14 +39,17 @@
     "regenerate:screenshots:rebuild": "npm run rm:builds && node scripts/rebuild-e2e.js",
     "rm:builds": "npm run rm:builds --workspace=scripts",
     "start": "npm run start --workspace=patternhub",
+    "test:lighthouse": "npm run build && npm run build --workspace=react-showcase && lhci autorun",
```

```suggestion
    "pretest:lighthouse": "npm run build && npm run build --workspace=react-showcase",
    "test:lighthouse": "lhci autorun",
```

this makes it a little more universal (I hope).

---

was a leftover, removed it